src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -13,11 +13,15 @@
 
 from .client import SlackClient
 from .link_identity import build_linking_url
+from .unlink_identity import build_unlinking_url
 from .requests import SlackActionRequest, SlackRequestError
 from .utils import build_group_attachment, logger
 
+
 LINK_IDENTITY_MESSAGE = "Looks like you haven't linked your Sentry account with your Slack identity yet! <{associate_url}|Link your identity now> to perform actions in Sentry through Slack."
 
+UNLINK_IDENTITY_MESSAGE = "Looks like this Slack identity is linked to a user that no longer exists! <{associate_url}|Unlink your identity now>."
```

Comment:
@NisanthanNanthakumar To be clear, this error does not mean the user doesn't exist. If the user did not exist, we'd get an error when trying to look up the linked identity. It means that the linked user is not part of an organization for that integration. I realize that the card for this had an incorrect explanation so I have updated the description (sorry for the confusion).

It might also be helpful if we could actually show the email of the user we used that's incorrect. Something like:

`It looks like this Slack identity is linked to the Sentry use <user_email> which is not a member of organization  <org_name> used with this Slack integration <{associate_url}|Unlink your identity now>.. `

Reply:
<img width="712" alt="Screen Shot 2020-07-07 at 11 43 18 AM" src="https://user-images.githubusercontent.com/10491193/86827620-170a8d80-c047-11ea-8827-1b878224a5ba.png">

Reply:
@NisanthanNanthakumar I didn't mean to literally include the `<` and `>` in the view the user sees lol

Reply:
But if you made those variables bold it might be nice (though definitely not necessary).

Reply:
<img width="702" alt="Screen Shot 2020-07-07 at 11 48 04 AM" src="https://user-images.githubusercontent.com/10491193/86828284-de1ee880-c047-11ea-8a8a-f8a1bc6bce10.png">

Reply:
@NisanthanNanthakumar Thanks for that change. You also need a period after `Slack integration`.